tests/api/test_bulk_delete.py

```diff
@@ -286,6 +298,48 @@ def test_delete_draft_orders(
         id__in=[order.id for order in orders]).count() == len(orders)
 
 
+def test_fail_to_delete_non_draft_order_lines(
+        staff_api_client, order_with_lines, permission_manage_orders):
+    order = order_with_lines
+    order_lines = [line for line in order]
+    # Set anything but OrderStatus.DRAFT
+    order.status = OrderStatus.CANCELED
+    order.save()
+
+    variables = {'ids': [
+        graphene.Node.to_global_id('OrderLine', order_line.id)
+        for order_line in order_lines]}
+    response = staff_api_client.post_graphql(
```

Comment:
```suggestion

    # Post the invalid request and 
    # ensure the correct permissions are required
    response = staff_api_client.post_graphql(
```

Reply:
I don't think this comment is necessary, as we are doing it for almost every request that requires permission(s).

As a side note, I would like to note, it slows us down IMO, as we are doing 2 requests, one which checks that user has no permission and second, that adds the appropriate one. It is tedious to use too. Maybe we can talk with @maarcingebala about this in the near future.